enhancements/security/openshift-image-policy.md

```diff
@@ -0,0 +1,359 @@
+
+## Graduation Criteria
+
+This functionality is expected to use [the existing `ImagePolicy` feature gate][feature-gate], and have the same graduation criteria.
+
+### Dev Preview -> Tech Preview
+
+[The existing `ImagePolicy` feature gate is already `TechPreviewNoUpgrade`][feature-gate], so this transition will never happen for this new functionality.
+
+### Tech Preview -> GA
+
+This functionality is expected to use [the existing `ImagePolicy` feature gate][feature-gate], and have the same graduation criteria.
+
+### Removing a deprecated feature
+
+This proposal does not deprecate any features.
+This proposal does not remove any features, with the possible exception of the removal of the OCP image scope guard discussed in [the *API Extensions* section](#api-extensions).
+
+## Upgrade / Downgrade Strategy
+
+Existing clusters can return to the previous behavior after updating by setting the `overrides` entry mentioned in [the *Installer* section](#installer) and removing the policy with:
+
+```console
+$ oc delete clusterimagepolicy openshift
+```
+
+## Version Skew Strategy
+
+The new ClusterImagePolicy manifest should have a late enough run-level that the cluster-version operator (CVO) applies it after the ClusterImagePolicy CRD and any machine-config tooling needed to process the custom resource.
+Because the machine-config is at run-level 80, late in the update, this means that updates into new image policies will be applied after most other components have rolled out their new images, which may require slower ratcheting for things like signing key rotations.
+
+## Operational Aspects of API Extensions
+
+None of the changes proposed in [the *API Extensions* section](#api-extensions) have operational impact.
+
+## Support Procedures
+
+I expect "images are failing to meet the relevant image policy/policies" to be something with a solid support experience, regardless of whether OpenShift defines its own image policy, or leaves image policies entirely up to users.
+
+The process for disabling the image policy is described in [the *Upgrade / Downgrade Strategy* section](#upgrade--downgrade-strategy).
+
+## Alternatives
+
+### Continue with OpenPGP signatures
+
+We could continue to use OpenPGP signatures, and save the trouble of building a new process and tooling.
+But our OpenPGP image-signing system is largely custom, and that has more maintenance costs than a more standardized, upstream process.
+The OpenPGP signatures are also not stored in registries, which makes [mirroring more complicated][OTA-946].
+The OpenPGP signatures also only covered the root release image, requiring a chain of trust based on image digests to extend the trust to referenced payload images like the machine-config operator.
```

Comment:
Another alternative is to start shipping OpenPGP signatures for all images in the release payload.

But, for the most part, that would look just like this enhancement — we would either need an `ClusterImagePolicy`, or an equivalent MCO configuration payload enforcing the signature validity; and most of the effort would be around coordinating/managing upgrades, policy changes and mirroring, just like in the primary proposal.